README.md

```diff
@@ -1,6 +1,6 @@
 # 💳🌽 Over-Collateralized Lending
 
-![readme-lending](https://raw.githubusercontent.com/scaffold-eth/se-2-challenges/over-collateralized-lending/extension/packages/nextjs/public/hero.png)
+![readme-lending](https://raw.githubusercontent.com/scaffold-eth/se-2-challenges/challenge-over-collateralized-lending/extension/packages/nextjs/public/hero.png)
 
 💳 Build your own lending and borrowing platform. Let's write a contract that takes collateral and lets you borrow other assets against the value of the collateral. What happens when the collateral changes in value? We will be able to borrow more if it is higher or if it is lower we will also build a system for liquidating the debt position.
```

Comment:
> We will be able to borrow more if it is higher or if it is lower we will also build a system for liquidating the debt position.

maybe add some commas here like: `We will be able to borrow more if it is higher, or if it is lower, we will also build a system for liquidating the debt position.`